wof/examples/production_configs/wofpy.conf

```diff
@@ -15,5 +15,5 @@ setgid www-data
 env PATH=/home/ubuntu/miniconda/envs/wofpy/bin
 
 # change directory to where the uWSGI configuration file is located then execute uwsgi with the config file
-chdir /var/www/timeseries
-exec uwsgi --ini timeseries.ini
\ No newline at end of file
+chdir /var/www/wofpy
+exec uwsgi --ini wofpy.ini
```

Comment:
> Got it! So this means that a user who wants to run more than one service would still only run that single line systemctl start wofpy?!

Yes

> It looks like the file wofpy would also need to be modified for a multi-service set up, no?

Yep. Hence the comment in [L13](https://github.com/lsetiawan/WOFpy/blob/533f6d3bffdf26b987da7c2e8b3a45c0107759cf/wof/examples/production_configs/wofpy#L13) Just gotta add the URL Configurations for multiple dao's